CMakeLists.txt

```diff
@@ -113,6 +114,7 @@ if("${ARCH}" MATCHES "s390x")
     option(WITH_DFLTCC_DEFLATE "Use DEFLATE COMPRESSION CALL instruction for compression on IBM Z" OFF)
     option(WITH_DFLTCC_INFLATE "Use DEFLATE COMPRESSION CALL instruction for decompression on IBM Z" OFF)
 endif()
+add_feature_info(WITH_CVES WITH_CVES "Build test/CVEs")
```

Comment:
This doesn't look like best place for this line...

Reply:
I put it here because it should merge in nicely after #352.

Reply:
The original message on the pull request doesn't mention this pull request depends on #352.

Reply:
Ok I've updated it.